scripts/google_entities.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+# Update current Google entity IDs using ProPublica Congress API.
+
+import json, urllib.request, urllib.parse, urllib.error
+import rtyaml
```

Comment:
The build is failing because this import statement is unused. I'll fix this on-branch myself.